recipes/openblas/all/conanfile.py

```diff
@@ -46,6 +50,12 @@ def _configure_cmake(self):
 
         cmake.definitions["NOFORTRAN"] = not self.options.build_lapack
         cmake.definitions["BUILD_WITHOUT_LAPACK"] = not self.options.build_lapack
+        cmake.definitions["DYNAMIC_ARCH"] = self.options.dynamic_arch
+        cmake.definitions["USE_THREAD"] = self.options.use_thread
```

Comment:
`package_info` already adds `pthread`, but shouldn't it now be added conditionally?

Reply:
Yes. I'll fix that and update the pull request quickly.